_posts/2020-01-08-cockpit-210.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Cockpit 210 and Cockpit-podman 12
+author: kkoukiou
+date: 2020-01-08
+tags: cockpit linux
+slug: cockpit-210
+category: release
+summary: Cockpit with Overview, Dashboard and Machines improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes
+from version 210.
+
+### Overview: Add CPU utilization to usage card
+
+Display CPU usage information in the usage card of the Overview page. For detailed CPU usage information, please
+consult the graphs page.
+
+![Overview](/images/overview-cpu-usage.png)
+
+### Dashboard: Allow adding locked identity to keyring when adding new machines
+
+Cockpit now offers an inline feature to unlock SSH identities when adding a new machine to the Dashboard.
+
+![unlock-keyring](/images/unlock-keyring.png)
+
+### SElinux: Introduce an Ansible automation script
+
+SELinux modifications can reviewed and exported as Ansible playbooks. These exported playbooks can be
+applied to other servers.
+
+![SElinux autoscript ansible](/images/selinux-autoscript-ansible.png)
+
+### Machines: Support "bridge" type network interfaces
+
+Allow setting up Network Interfaces which provide a bridge from the VM directly to the LAN. This provides the guest VM
+full incoming and outgoing network access just like a physical machine.
+
+![VM NIC bridge](/images/vm-nic-bridge.png)
+
+### Machines: Support "bus" type configuration for disks
+
+When adding new or editing existing VM disks allow the user to choose between different bus types, such as SATA, SCSI,
+USB or VIRTIO.
+
+[Disk choose bus type](/images/disk-choose-bus-type.png)
+
+### Podman: Configure CPU share for system containers
```

Comment:
```suggestion
### Podman: System container CPU sharing
```

This might be better?

Reply:
As discussed over IRC CPU share shall be kept as is because the documentation always mentions it like this as well https://docs.docker.com/engine/reference/run/